discovery-infra/start_discovery.py

```
@@ -752,13 +759,24 @@ def is_none_platform_mode():
     return args.platform.lower() == consts.Platforms.NONE
 
 
-def main():
-    cluster_name = f'{args.cluster_name or consts.CLUSTER_PREFIX}-{args.namespace}'
-    log.info('Cluster name: %s', cluster_name)
+def try_get_cluster():
+    try:
+        if args.cluster_id:
+            client = assisted_service_api.create_client(
+                url=utils.get_assisted_service_url_by_args(args=args)
+            )
+            return client, client.cluster_get(cluster_id=args.cluster_id)
 
+    except ApiException:
+        pass
```

Please log, don't ignore errors